Formula/postgresql.rb

```diff
@@ -100,18 +100,20 @@ def post_install
   end
 
   def caveats; <<-EOS.undent
-    If builds of PostgreSQL 9 are failing and you have version 8.x installed,
+    If builds of PostgreSQL 10 are failing and you have version 8.x installed,
```

Comment:
`8.x` -> `9.x`? May need to check if is relevant at all

Reply:
I never had an issue when installing a newer postgres using `brew upgrade`.
The issue linked indicates that there used to be a problem when upgrading from 8.4.4 to 9.0.

I'm happy to remove the comment completely.

Reply:
the fewer caveats the better